Source/SwiftLintFramework/Models/LinterCache.swift

```diff
@@ -15,7 +15,7 @@ public enum LinterCacheError: Error {
     case differentConfiguration
 }
 
-public struct LinterCache {
+public class LinterCache {
```

Comment:
I had to change it to be a class to make sure all `Linter`s used the same reference